lib/logstash/outputs/http.rb

```diff
@@ -80,6 +81,9 @@ class LogStash::Outputs::Http < LogStash::Outputs::Base
   # Otherwise, the event is sent as json.
   config :format, :validate => ["json", "form", "message"], :default => "json"
 
+  # Set this to true if you want to enable gzip compression for your message
+  config :compression, :validate => :boolean
```

Comment:
Can we rename this option to `http_compression` this will make sure we the same naming convention as our elasticsearch output, see the doc at https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-http_compression